llvm/lib/Transforms/Yk/ModuleClone.cpp

```diff
@@ -78,8 +78,16 @@ struct YkModuleClone : public ModulePass {
       if (F.hasExternalLinkage() && F.isDeclaration()) {
         continue;
       }
+      // Function with address taken are not cloned.
```

Comment:
Can we add a reason for why these are not cloned? While we are here there's some typos here: function**s**, has **its** address taken.

Reply:
Added explanation :point_right: c84aec1db6b4fc57d2a2a28d5531574111118ad3